addons/kafka/templates/cmpd-broker.yaml

```diff
@@ -228,9 +230,10 @@ spec:
       - name: jmx-exporter
         image: {{ .Values.images.registry | default "docker.io" }}/{{ .Values.images.jmxExporter.repository }}:{{ .Values.images.jmxExporter.tag }}
         imagePullPolicy: {{ default "IfNotPresent" .Values.images.pullPolicy }}
+        {{- if .Values.container.securityContext }}
         securityContext:
-          runAsNonRoot: true
```

Comment:
The modification logic here differs from the previous implementation. 

the default value of allowPrivilegeEscalation is true in k8s？

```
	// defaultAllowPrivilegeEscalation controls the default setting for whether a
	// process can gain more privileges than its parent process.
	// +optional
	DefaultAllowPrivilegeEscalation *bool `json:"defaultAllowPrivilegeEscalation,omitempty" protobuf:"varint,15,opt,name=defaultAllowPrivilegeEscalation"`
	// allowPrivilegeEscalation determines if a pod can request to allow
	// privilege escalation. If unspecified, defaults to true.
	// +optional
	AllowPrivilegeEscalation *bool `json:"allowPrivilegeEscalation,omitempty" protobuf:"varint,16,opt,name=allowPrivilegeEscalation"`
```

Reply:
Yes, so in the previous implementation, the jmx-exporter will be able to `sudo`. Now it can't.

If we jmx-exporter needs `AllowPrivilegeEscalation`, i can define another value for it.